JustSaying/AwsTools/MessageHandling/SqsPublisher.cs

```diff
@@ -40,13 +41,15 @@ public void Publish(Message message)
         }
 #endif
 
-        public async Task PublishAsync(Message message)
+        public async Task PublishAsync(Message message) => await PublishAsync(message, CancellationToken.None);
```

Comment:
Could you just `return` this please. (or use `.ConfigureAwait(false)`, but I'd rather it just returned the task)